cmd/minishift/cmd/start.go

```diff
@@ -255,13 +270,15 @@ func determineInsecureRegistry(key string) []string {
 	return append(s, defaultInsecureRegistry)
 }
 
-func startHost(libMachineClient *libmachine.Client) (*host.Host, string) {
+func startHost(libMachineClient *libmachine.Client, vmDriver string) *host.Host {
```

Comment:
Not quite sure whether I'd add the driver parameter here. Why not retrieve it again via Viper. Just wondering?